boards/DJI_Board_TypeA/Middlewares/ST/STM32_USB_Device_Library/Class/CDC/Src/usbd_cdc.c

```diff
@@ -446,8 +446,6 @@ __ALIGN_BEGIN static uint8_t USBD_CDC_OtherSpeedCfgDesc[USB_CDC_CONFIG_DESC_SIZ]
   * @{
   */
 
```

Comment:
This part is specifically modified by Alvin to resolve USB bug presented in Middleware. This file should not be changed.